core/src/test/java/dev/streamx/cli/ingestion/IngestionArgumentsValidationTest.java

```diff
@@ -114,14 +64,7 @@ public void shouldRejectKeylessIngestion(QuarkusMainLauncher launcher) {
         Error: Missing required argument(s): <key>
         Usage: streamx unpublish [-hV] [[--ingestion-url=<propagateIngestionUrl>]]
                                  (<channel> <key>)
-        Send unpublication trigger
-              <channel>   Channel that message will be published to
-              <key>       Message key
-          -h, --help      Show this help message and exit.
-              --ingestion-url=<propagateIngestionUrl>
-                          Address of 'rest-ingestion-service'
-                            Default: http://localhost:8080
-          -V, --version   Print version information and exit.""");
+        Try 'streamx unpublish --help' for more information.""");
```

Comment:
If it's possible - add newline to separate suggestion "Try" from rest of information.

Reply:
Done